### ISSUE TITLE
Question: would you like to discuss adding libsndfile to vcpkg?

### ISSUE THREAD

**Issue by bagong** [erikd/libsndfile#253]
I have a "port" running which works as expected, it's almost trivial to add libsndfile to vcpkg due to the good cmake support now. The only thing missing is the generation of a pcb-symbol file. I am not sure yet how to get that yet.

I could just go ahead and see which feedback I get over there. But you might want to have a word in it? The way how libsndfile get's added to vcpkg - which will happen sooner or later anyways - will influence if further development of the package goes more through vcpkg itself (where people are quick to patch stuff themselves) or through "upstream". (As an example, the flac and vorbis ports in vcpkg are quite significantly patched).

So if wished, I'll post my portfile here and we can discuss what to change...

**Comment by evpobr** [comment]
As I said before, you do not need to ask permission for a derivative work from our library. Updating any obsolete ports and packages is welcome, thanks for that. But this is not a topic of discussion in the libsndfile repository. If you run into problems with maintraining packages that need to be fixed in the source code, then create issue or pull request here.

So just make pull request to vcpkg and and libsndfile port. I follow that repo and will be happy to see libsndfile package there.

Also, please continue testing our library under Windows. I found a lot of bugs with your help. The libsndfile community is small and any help will be useful.

**Comment by erikd** [comment]
Again, this is not really any of my business. I primary wrote libsndfile for myself and for the Linux community. I also considered it advantageous to support other operating systems as well as I could. Windows has always been a huge pain in the neck. The amount of time and effort I have spent supporting libsndfile on Windows is easily 10 times what I have spent supporting OS X, FreeBSD, NetBSD, OpenBSD, Solaris and all the rest put togther. I also don't use Windows myself and honestly I don't understand why anyone else does either.

As long as you abide by the license you are free to do whatever you want, and you do not have to tell me about it. And if you have sane patches, I'd be happy to receive them.
